stablehlo/reference/Element.cpp

```diff
@@ -714,7 +905,7 @@ Element power(const Element &e1, const Element &e2) {
       if (base.abs().isOne())
         exponent = exponent.abs();
       else
-        return Element(type, static_cast<int64_t>(0));
+        return convert(type, 0L);
```

Comment:
It's a good idea to simplify `static_cast<int64_t>(n)` to `nL`. Let's also apply it to other occurrences below.